doc/man3/flux_watcher_set_priority.rst

```diff
@@ -0,0 +1,43 @@
+============================
+flux_watcher_set_priority(3)
+============================
+
+.. default-domain:: c
+
+SYNOPSIS
+========
+
+.. code-block:: c
+
+   #include <flux/core.h>
+
+   flux_watcher_t *flux_watcher_set_priority (flux_watcher_t *w,
+                                              int priority);
+
+Link with :command:`-lflux-core`.
+
+DESCRIPTION
+===========
+
+:func:`flux_watcher_set_priority` sets the priority on the watcher.
+Higher priority watchers run first.  The range of priorities is from
+-2 to 2, with the default being 0.  If the priority is out
+of range, the max or min value is set.  The priority should only be
+set when the watcher is stopped.
```

Comment:
The libev manual makes a stronger statement here:

> You _must not_ change the priority of a watcher as long as it is active or pending. 

I'm wondering if that's warranted here as well? I don't actually know the effects.

Reply:
Eh, it's probably OK as is IMHO.  I would assume it just wouldn't have any effect since the watchers are likely in stored in a list per priority level, inserted when started and removed when stopped.